app/src/main/index.js

```diff
@@ -308,10 +336,14 @@ function stopLCD() {
     try {
       // prevent the exit to signal bad termination warnings
       lcdProcess.removeAllListeners(`exit`)
-      lcdProcess.on(`exit`, resolve)
+
+      lcdProcess.on(`exit`, () => {
+        lcdProcess = null
```

Comment:
what is the thinking re: `null` instead of `false`?

Reply:
false would mix different types for one variable (Promise, Boolean). in JS everything is possible. but it's better to create predictable code I think.

Reply:
if i understand you correctly... you are saying `null` and `true` are required - adding `false` to the mix would be unnecessary complication?

Reply:
`false` is traditionally used for boolean propositions (e.g., "There is an `lcdProcess` running.").

`null` is used to communicate absence of value ("`lcdProcess` contains an instance of `ChildProcess` and in this case there isn't one.")